dist/stat_data.py

```diff
@@ -1004,6 +1004,7 @@ def __init__(self, name, desc, flags=''):
     SessionStat('bytes_read', 'bytes read into cache'),
     SessionStat('bytes_write', 'bytes written from cache'),
     SessionStat('cache_time', 'time waiting for cache (usecs)'),
+    SessionStat('evict_help_skipped', 'number of times eviction was skipped for the transaction'),
```

Comment:
Maybe `number of times application eviction was skipped`? 

I think it will be confusing to just call this `eviction` as people could confuse it with the other stats that use the term "eviction" more generally. E.g., "leaf pages split during eviction". So `application eviction` seems better. 

`for the transaction` doesn't really add anything for me and is a bit confusing. Which transaction is *the* transaction? I suppose maybe it could say `for a transaction` or `during transaction API call`

Reply:
I added 'application'. I'm also not happy with 'skipped' as it implies the code made a choice to skip it based on some conditions. Maybe 'turned off'?